doc/classes/Vector4.xml

```diff
@@ -177,7 +177,7 @@
 		<method name="min_axis_index" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the axis of the vector's lowest value. See [code]AXIS_*[/code] constants. If all components are equal, this method returns [constant AXIS_W].
+				Returns the axis of the vector's lowest value. See [code]AXIS_*[/code] constants. If all components are equal, this method returns [constant AXIS_X].
```

Comment:
Actually, that's a bug. `min_axis_index` should be adjusted to have the same behavior as Vector2 and Vector3.

Reply:
should I adjust Vector4 and Vector4i accordingly? and in this PR?

Reply:
I think it would make sense to have the fix in its own PR, but it's fine either way.

Reply:
Hm… ok, then I'll go with a new PR. Makes for cleaner history

Reply:
Wait… `min_axis_index` returns x for Vector 2 and 3. It's always different for `max_axis_index` -> here it's always the highest letter.

Reply:
OK, now I get it, it returns W for Vector4 and Vector4i because of `<=` -> gets overwritten in every loop…
```c++
	uint32_t min_index = 0;
	int32_t min_value = x;
	for (uint32_t i = 1; i < 4; i++) {
		if (operator[](i) <= min_value) {
			min_index = i;
			min_value = operator[](i);
		}
	}
	return Vector4i::Axis(min_index);
```

So no bug. And docs were right. I just didn't expect this logic… I only overwrite my min value if it's actually smaller…

Reply:
This is why it's very important to test the code :)